src/address.js

```diff
@@ -42,29 +42,39 @@ export function getPickupAddress(pickupSla) {
   )
 }
 
+export function isCurrentAddressType(address, addressType) {
+  if (address && !address.addressType) return false
+
+  if (address && (typeof address.addressType === 'string')) {
+    return address.addressType === addressType
+  }
+  return address && address.addressType.value === addressType
+}
+
 /** PUBLIC **/
 
 export function isAddressComplete(address) {
   const { number, street, neighborhood, city, state } = address || {}
   return !!(number && street && neighborhood && city && state)
 }
 
+export function isGiftRegistry(address) {
```

Comment:
Cirou a função nova de `isGiftRegistry` e não criou o teste, estamos de olho

Reply:
Sorry for the vacilo.